Fix Bitbucket source url

carlosdevpereira opened the pull request:
@qinezh  Bitbucket repository url's use "src" instead of "blob" :)

Commit:
Fix Bitbucket source url

Bitbucket repository url's use "src" instead of "blob" :)

qinezh reviewed (commented):
Thanks your contribution @carlosdevpereira ,but seems it not works as expected, have you tested it?

Commit:
Update linkInfo.ts

Ohh sorry :O, i was really high when i wrote that :D, it should be fixed now...

qinezh commented on src/linkInfo.ts:
```diff
@@ -13,11 +13,12 @@ export class LinkInfo {
         var startIndex = this.position.start.line + 1;
         var endIndex = this.position.end.line + 1;
-        var prefix = `https://${this.siteName}/${this.userName}/${this.repoName}/blob/${this.branchName}/${this.filePath}`;
+        var siteUrlSource = (this.siteName.split("@")[1] === "bitbucket.org") ? 'src' : 'blob';
```
why need `split("@")` here?

carlosdevpereira replied:
I debugged the extension within vscode and siteName was returning something like myUserName@siteDomain, (where myUserName is your git username and siteDomain is the remote domain), and to compare if the remote URL was from bitbucket or from github i splitted siteName to compare only the domain...

carlosdevpereira replied:
basically for bitbucket repositories siteName returns myUserName@siteDomain, and for github repositories it only returns siteDomain, either way, i did this way to work on both.. :)

qinezh replied:
Got it. I'll merge it and release a new version around these days. Thanks~

qinezh commented:
@carlosdevpereira the new version of GitLink has been released.

carlosdevpereira commented:
@qinezh Ok thanks. 👍
